src/components/common/TmSessionImport.vue

```diff
@@ -81,7 +86,10 @@ export default {
     }
   },
   validations: () => ({
-    seed: { required, words24 }
+    seed: {
+      required,
+      validSeed: param => words24(param) || polkadotRawSeed(param)
```

Comment:
In the future I guess this will need to be refactored to a dictionary that takes the network_type property in network and then selects the proper seed function

Reply:
we will also need to change the wording and error messages

Reply:
rn this is just for us